README.md

```diff
@@ -27,16 +29,40 @@ npm start
 
 Note that data is stored on disk for persistence under `volumes` in the compose file.
 
-## Set up a local mongoDB container
+## Running multiple node servers
+Alternatively, you can also use concurrently for backend development to run multiple node servers. This would be suitable if you're focusing on backend developement. 
+
+Ensure that either a mongo server is up or the connection string points to Atlas. See below for instructions on setting up a local mongo container.
+
+First you have to create a `.env` file in the root folder. It should look like the following:
+
+```
+DB_CLOUD_URI=
+DB_LOCAL_URI=mongodb://localhost:27017/cs3219
+DB_TEST_URI=mongodb://localhost:27017/userTests
+SECRET=

```

Comment:
Maybe can add a note/comment under the env example like "add DB_CLOUD_URI if you want to connect to mongodb cloud....SECRET if you want to set the secret variable..."

or

add `<Placeholder>` for DB_CLOUD_URI and SECRET 
eg.  SECRET=`<secret phrase for token encryption>` DB_CLOUD_URI=`<MongoDB cloud connection string>`